css/css-ui/tools/appearance-build-webkit-reftests.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python3
```

Comment:
Can you run this script in https://github.com/web-platform-tests/wpt/blob/master/update-built-tests.sh to make sure committed output doesn't diverge from script output?

Reply:
Fixed